src/pages/Login/Login.js

```diff
@@ -1,7 +1,170 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { AlertCircle, ArrowLeft, Eye } from 'react-feather';
+import './Login.scss';
 
 function Login() {
-  return <div>Login.js</div>;
+  const [inputs, setInputs] = useState({
+    userId: '',
+    password: '',
+  });
+
+  const { userId, password } = inputs;
+
+  const navigate = useNavigate();
+
+  const [passwordVisible, setPasswordVisible] = useState(false);
+
+  const [warning, setWarning] = useState({
+    idIncorrect: false,
+    pwIncorrect: false,
+  });
+
+  const { idIncorrect, pwIncorrect } = warning;
+
+  const handleInput = event => {
+    const { name, value } = event.target;
+    event.preventDefault();
+
+    setInputs(prev => ({
+      ...prev,
+      [name]: value,
+    }));
+  };
+
+  const handleWarningMsg = (validation, item) => {
+    validation
+      ? setWarning({ ...warning, [item]: false })
+      : setWarning({ ...warning, [item]: true });
```

Comment:
```suggestion
      setWarning({ ...warning, [item]: !validation })
```
validation의 반대가 들어가는 거니 이렇게 구현할 수 있겠네요!